mysql-test/suite/rpl/t/rpl_percona_bug995624.test

```diff
@@ -0,0 +1,46 @@
+source include/master-slave.inc;
+source include/not_gtid_enabled.inc;
```

Comment:
If this check is required, then the cheap testcase-skipping checks should go before the expensive testcase setup actions (master-slave.inc), or MTR will go through the trouble of starting a slave server, configuring replication, only to see that the test is going to be skipped in some cases